lib/CMakeLists.txt

```diff
@@ -5,3 +5,5 @@ else()
 endif()
 
 add_dependencies(OptSched ${OPT_SCHED_TARGET_DEPS})
+
+target_link_libraries(OptSched -L/home/jeff/ROCm/opencl/build/lib/ libamdocl64.so)
```

Comment:
Needs to be removed for the PR.

However, note that you can do this without modifying the CMakeLists.txt now: when you configure with CMake (or reconfigure), add `-DOPTSCHED_EXTRA_LINK_LIBRARIES=/path/to/ROCm/opencl/build/lib/libamdocl64.so`

Reply:
Thanks for the tip!